### ISSUE TITLE
HA Sending up instability issues.

### ISSUE THREAD

**Issue by Humphaz** [rsnodgrass/hass-sensorpush#5]
Logger: homeassistant.util.async_
Source: util/async_.py:120
First occurred: 10:14:14 PM (3 occurrences)
Last logged: 10:14:15 PM

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for sensorpush doing I/O at custom_components/sensorpush/sensor.py, line 44: LOG.debug(f"Setting up SensorPush sensors: {sensorpush_service.sensors}")

**Comment by tehbra1n** [comment]
I've been seeing the same since this new warning was announced as a feature by HA one or two revs back.  It's present in the current beta 0.111b1.

```
2020-06-04 17:30:51 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for sensorpush doing I/O at custom_components/sensorpush/sensor.py, line 44: LOG.debug(f"Setting up SensorPush sensors: {sensorpush_service.sensors}")
2020-06-04 17:30:51 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for sensorpush doing I/O at custom_components/sensorpush/sensor.py, line 44: LOG.debug(f"Setting up SensorPush sensors: {sensorpush_service.sensors}")
2020-06-04 17:30:51 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for sensorpush doing I/O at custom_components/sensorpush/sensor.py, line 44: LOG.debug(f"Setting up SensorPush sensors: {sensorpush_service.sensors}")
```

**Comment by rsnodgrass** [comment]
Thanks for the heads up.  pysensorpush doesn't support asynch I/O, so I've changed hass-sensorpush sensor.py to not be asynchronous (until pysensorpush eventually has an async API).  Change is in master, and I'm testing it now.

**Comment by rsnodgrass** [comment]
Fixed in release 0.0.5
